index.html

```diff
@@ -50,6 +58,203 @@ <h1>Hey there.
+                        <h1 class="cardtitle">Profesional Art Printing Data</h1>
+                        <p class="cardsupporting">
+                            A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required. 
+                            has been the industry's standard
+                        </p>
+                        <ul class="tags catags">
+                            <li class="taging cataging"><p class="tag catag">html</p></li>
+                            <li class="taging cataging"><p class="tag catag">bootstrap</p></li>
+                            <li class="taging cataging"><p class="tag catag">Ruby</p></li>
+                        </ul>
+                    </section>
+                    <div class="buttn">
+                        <button type="button">See Project</button> 
+                    </div> 
+                </div> 
+
+                <div class="cards">
+                    <section class="rightblock">
+                        <h1 class="cardtitle">Profesional Art Printing Data</h1>
+                        <p class="cardsupporting">
+                            A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required. 
+                            has been the industry's standard
+                        </p>
+                        <ul class="tags catags">
+                            <li class="taging cataging"><p class="tag catag">html</p></li>
+                            <li class="taging cataging"><p class="tag catag">bootstrap</p></li>
+                            <li class="taging cataging"><p class="tag catag">Ruby</p></li>
+                        </ul>
+                    </section>
+                    <div class="buttn">
+                        <button type="button">See Project</button> 
+                    </div> 
+                </div> 
+
+                <div class="cards">
+                    <section class="rightblock">
+                        <h1 class="cardtitle">Profesional Art Printing Data</h1>
+                        <p class="cardsupporting">
+                            A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required. 
+                            has been the industry's standard
+                        </p>
+                        <ul class="tags catags">
+                            <li class="taging cataging"><p class="tag catag">html</p></li>
+                            <li class="taging cataging"><p class="tag catag">bootstrap</p></li>
+                            <li class="taging cataging"><p class="tag catag">Ruby</p></li>
+                        </ul>
+                    </section>
+                    <div class="buttn">
+                        <button type="button">See Project</button> 
+                    </div> 
+                </div> 
```

Comment:
- [ ] Great job so far, But I can see here that you wrap the section inside the div you don't need the div just put the section instead of the div with the same class name because it is more recommended to use a section tag because `section` means that the content inside is grouped (i.e. relates to a single theme), and should appear as an entry in an outline of the page. but `div` , on the other hand, does not convey any meaning, aside from any found in its class, lang and title attributes. So no: using a `div` does not define a section in HTML. so please update it👌🏻